lib/wit/main.py

```diff
@@ -325,9 +325,9 @@ def status(ws, args) -> None:
         for path in untracked:
             relpath = path.relative_to(ws.root)
             log.info("    {}".format(relpath))
-    if len(damaged) > 0:
-        log.info("Damaged packages:")
-        for package in damaged:
+    if len(missing) > 0:
+        log.info("Missing packages:")
+        for package in missing:
```

Comment:
Is renaming it right? Doesn't the category "damaged" include other possible problems?

Reply:
No, currently they are the same thing.
The cleanest thing to do would be to call Package.status to get each package's (category, labels) (e.g. `("Damaged", ["missing"])`) but I'm not sure if you want that in this PR

Reply:
Okay I see. Yeah it's fine to do that later